src/controller/java/AndroidDeviceControllerWrapper.h

```diff
@@ -0,0 +1,65 @@
+/*
+ *   Copyright (c) 2020 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+#pragma once
+
+#include <memory>
+
+#include <jni.h>
+
+#include <controller/CHIPDeviceController.h>
+
+#include "AndroidDevicePairingDelegate.h"
+
+/**
+ * This class contains all relevant information for the JNI view of CHIPDeviceController
+ * to handle all controller-related processing.
+ *
+ * Generally it contains the DeviceController class itself, plus any related delegates/callbacks.
+ */
+class AndroidDeviceControllerWrapper
+{
+public:
+    ~AndroidDeviceControllerWrapper();
+
+    chip::DeviceController::ChipDeviceController * Controller() { return mController.get(); }
```

Comment:
is upper-case here intentional (since all remaining method names start with lower-case)? Also could the method be const?

Reply:
The return value is non-const, so I made the method itself non-const as well (even if it probably could be const).

I updated all methods to use uppercase now.